PPP_Robot:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50 

  z_axis_controller:
    type: position_controllers/JointPositionController
    joint: z_axis
    pid: {p: 100.0, i: 0.01, d: 10.0}

  y_axis_controller:
    type: position_controllers/JointPositionController
    joint: y_axis
    pid: {p: 1000.0, i: 0.11, d: 1.0}

  x_axis_controller:
    type: position_controllers/JointPositionController
    joint: x_axis
    pid: {p: 100.0, i: 0.11, d: 1.0}
